lib/publiccloud/instance.pm

```diff
@@ -961,22 +985,18 @@ sub upload_supportconfig_log {
     my $start = time();
     my $logs = "/var/tmp/scc_supportconfig";
     # Eventual comma-separated tokens list to exclude
+    # Excluding AUDIT due to bsc#1250310
     my $exclude = get_var('PUBLIC_CLOUD_SUPPORTCONFIG_EXCLUDE', 'AUDIT');
     # To remove exclusions, _EXCLUDE='-'
     $exclude = undef if ($exclude eq '-');
     $exclude = "-x " . $exclude if ($exclude);
-    # poo#187440 Workaround applied inject newline in ssh supportconfig to prevent hang cases, while bsc#1250310 open
-    my $cmd = "echo | sudo timeout --foreground --kill-after 60s $timeout supportconfig -R " . dirname($logs) . " -B supportconfig $exclude > $logs.txt 2>&1";
-    local $@;
-    my $res = eval { $self->ssh_script_run($cmd, timeout => ($timeout + 120)) };
-    if (isok($res) && !$@) {
-        $self->ssh_script_run(cmd => "sudo chmod 0644 $logs.txz");
-        $self->upload_log("$logs.txz", failok => 1, timeout => 180);
+    my $cmd = "echo | sudo supportconfig -R " . dirname($logs) . " -B supportconfig $exclude > $logs.txt 2>&1";
```

Comment:
In this case `supportconfig` might really run for veeery long.
I think it would make more sense to wrap it like this:
```
ssh sut sudo timeout supportconfig
```
instead of killing `ssh`.